vhd/packer/windows-2019-vars.json

```diff
@@ -2,6 +2,6 @@
     "image_publisher": "MicrosoftWindowsServer",
     "image_offer": "WindowsServer",
     "image_sku": "2019-Datacenter-Core-smalldisk",
-    "image_version": "17763.2183.2109130127",
+    "image_version": "17763.2300.2111032328",
```

Comment:
why are we updating this here? Wouldn't this have gone in during the vhd build pr?

Reply:
I don't know why this is showing up as a diff.
master has the same values https://github.com/Azure/aks-engine/blob/master/vhd/packer/windows-2019-vars.json

Reply:
I think this may be a github ui issue.
I did a rebase / force push after opening the PR (because I noticed this same diff).